File: openapi-test-java-server-spring/testapi/api/CreatePetOnPetCreatedCallback.java
package testapi.api;

import java.lang.Void;
import testapi.model.Pet;

/** Callback handler for createPet - OnPetCreated
  * Runtime expression: {$request.body#/callbackUrl}
  */
public interface CreatePetOnPetCreatedCallback {
  /** Called when pet is created */
  Void onPetCreatedCallback(Pet body);
}

File: openapi-test-scala-server-spring/testapi/api/CreatePetOnPetCreatedCallback.scala
package testapi.api

import java.lang.Void
import testapi.model.Pet

/** Callback handler for createPet - OnPetCreated
 * Runtime expression: {$request.body#/callbackUrl}
 */
trait CreatePetOnPetCreatedCallback {
  /** Called when pet is created */
  def onPetCreatedCallback(body: Pet): Void
}